clients/admin-ui/src/home/HomeContent.tsx

```diff
@@ -0,0 +1,119 @@
+import { Center, Flex, SimpleGrid, Text } from "@fidesui/react";
+import Link from "next/link";
+import * as React from "react";
+import { useEffect, useState } from "react";
+
+import { useFeatures } from "~/features/common/features.slice";
+
+import { MODULE_CARD_ITEMS, ModuleCardKeys } from "./constants";
+import { ModuleCard } from "./types";
+
+const HomeContent: React.FC = () => {
+  const COLUMNS = 3;
+  const [list, setList] = useState<ModuleCard[]>(
+    MODULE_CARD_ITEMS.filter((item) =>
+      [
+        ModuleCardKeys.ADD_SYSTEMS,
+        ModuleCardKeys.CONFIGURE_PRIVACY_REQUESTS,
+      ].includes(item.key)
+    )
+  );
+  const { connectionsCount, systemsCount } = useFeatures();
+  const hasConnections = connectionsCount > 0;
+  const hasSystems = systemsCount > 0;
+
+  const getCardItem = (key: ModuleCardKeys) => {
+    const cardItem = MODULE_CARD_ITEMS.find((item) => item.key === key);
+    return cardItem;
+  };
+
+  useEffect(() => {
+    if (hasConnections || hasSystems) {
+      if (hasConnections) {
+        const card = getCardItem(ModuleCardKeys.REVIEW_PRIVACY_REQUESTS);
+        if (card && !list.includes(card)) {
+          setList((prev) => [...prev, card as ModuleCard]);
+        }
+      }
+      if (hasSystems) {
+        const card = getCardItem(ModuleCardKeys.MANAGE_SYSTEMS);
+        if (card && !list.includes(card)) {
+          setList((prev) => [...prev, card as ModuleCard]);
+        }
+      }
+    }
+  }, [hasConnections, hasSystems, list]);
```

Comment:
Normally a useEffect shouldn't depend on an object that it's going to modify via state, as that can create an infinite render loop, e.g.:
```
[foo, setFoo] = useState(0)
useEffect(() => {
  setFoo(foo + 1)
}, [foo])
``` 
=> infinite render loop.

I suppose you're getting around this with with the `list.includes()` calls, but really that's a symptom of the list not needing to be in the state (as above).

Reply:
Issue has been resolved.